tensornetwork/tests/network_operations_test.py

```diff
@@ -218,6 +218,13 @@ def test_reachable(backend):
   _ = [nodes[n][0] ^ nodes[n + 1][1] for n in range(len(nodes) - 1)]
   assert set(nodes) == tn.reachable(nodes[0])
 
+  edges = []
```

Comment:
Dont append this to an existing test, Instead, create a new test.

Also, I would highly recommend not creating an `Edge` object directly like this, as the `Node`s that the edge is trying connect won't exist in those `Node`s. Instead, you should create edges via the `tn.connect` operation. (You can also use the `^` symbol if you don't like typing.) See the other tests or our tutorial on `tensornetwork.readthedocs.io` for examples.

Reply:
thanks for your attention and kind comments.
a new commit pushed, please check it out.